src/NHibernate.Validator/Util/AllowsMultipleWithIntersectingTagsAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace NHibernate.Validator.Util
+{
+	/// <summary>
+	/// Specify if constraint attribute can be used multiple times with intersecting tags. (No tags, or same tags)
```

Comment:
Parenthesis is misleading I think. It should be `(No tags, or some tags in common.)